app/services/allocations/finder.rb

```diff
@@ -4,7 +4,7 @@ module Allocations
   class Finder
     attr_reader :filter_params, :search_params
 
-    ALLOCATION_INCLUDES = [:from_location, :to_location, moves: %i[profile]].freeze
+    ALLOCATION_INCLUDES = [from_location: :suppliers, to_location: :suppliers, moves: %i[from_location to_location court_hearings profile person]].freeze
```

Comment:
gonna test this on staging again for posterity, tested locally with a lot of data